CMakeLists.txt

```diff
@@ -310,6 +311,29 @@ endif ()
 set(MLPACK_INCLUDE_DIRS ${MLPACK_INCLUDE_DIRS} ${ARMADILLO_INCLUDE_DIRS})
 set(MLPACK_LIBRARIES ${MLPACK_LIBRARIES} ${ARMADILLO_LIBRARIES})
 
+# Download stb_image for image loading.
+if (DOWNLOAD_STB_IMAGE)
```

Comment:
I know this makes things a little more complicated, but we should try to search the system first for STB before downloading it, and just use the system installed version if possible.  For instance, it is available in Debian.  In any case, you can take a look at how `DOWNLOAD_ENSMALLEN` is only used if we can't find `ensmallen.hpp` on the system.  You could do the same for `stb_image.h`.

Reply:
Okay